src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -331,49 +330,6 @@ public void testGenerateStudentCourseJoinEmail() throws IOException {
     }
 
     @Test
-    public void testSystemCrashReportEmailContent() throws IOException {
-
-        ______TS("user is logged in and the error has message");
-
-        AssertionError error = new AssertionError("invalid parameter");
-        String requestMethod = "GET";
-        String requestUserAgent = "user-agent";
-        String requestPath = "/page/studentHome";
-        String requestUrl = "/page/studentHome/";
-        String requestParam = "{}";
-        UserType userType = new UserType("Actual user ABC");
-
-        EmailWrapper email =
-                new EmailGenerator().generateSystemErrorEmail(requestMethod, requestUserAgent, requestPath,
-                                                              requestUrl, requestParam, userType, error);
-
-        // The stack trace is different depending on the environment in which the test is run at.
-        // As a workaround, after the last common line, change all the stack trace to "..."
-        String lastCommonLineRegex =
-                "(?s)(at org\\.testng\\.TestRunner\\.run\\(TestRunner\\.java:621\\)\\s*)at.*?(\\s*</code>)";
-        String modifiedContent = email.getContent().replaceAll(lastCommonLineRegex, "$1...$2");
-        email.setContent(modifiedContent);
-
-        String subject = String.format(EmailType.ADMIN_SYSTEM_ERROR.getSubject(),
-                                       Config.getAppVersion(), error.getMessage());
-
-        verifyEmail(email, Config.SUPPORT_EMAIL, subject, "/systemCrashReportEmail.html");
```

Comment:
This HTML file should be fine to be removed. It can be found in the folder `src/test/resources/emails`.